_includes/header.html

```diff
@@ -94,10 +94,7 @@
     <div class="usa-nav__secondary">
       <div class="usa-nav__secondary-container">
         <ul class="usa-nav__secondary-links">
-          <li class="usa-nav__secondary-item">
-            <a href="{% if site.lang != 'en' %}https://secure.login.gov/{{site.lang}}/sign_up/enter_email{% else %}https://secure.login.gov/sign_up/enter_email{% endif %}">{% t nav.create_account %}</a>
```

Comment:
Should/can we remove the "nav.create_account" translations now?

Reply:
Yep, the latest commit removed them. Thx for pointing it out.